build-scripts/config_common.cmake

```diff
@@ -296,6 +296,10 @@ if (WAMR_BUILD_JIT_STACK_FRAME EQUAL 1)
   add_definitions (-DWASM_ENABLE_JIT_STACK_FRAME=1)
   message ("     JIT stack frame enabled")
 endif ()
+if (WAMR_BUILD_GC_PERFORMANCE_PROFILING EQUAL 1)
+  add_definitions (-DWASM_ENABLE_GC_PERFORMANCE=1)
```

Comment:
Suggest using `WAMR_BUILD_GC_PERF_PROFILING` and `WASM_ENABLE_GC_PERF_PROFILING`